File: IzvorniKod/backend/humanitarni-setaci-pasa/src/main/java/hr/fer/progi/oneclick/humanitarnisetacipasa/models/StatisticsModel.java
package hr.fer.progi.oneclick.humanitarnisetacipasa.models;

import hr.fer.progi.oneclick.humanitarnisetacipasa.services.StatisticsService;

public class StatisticsModel {

    private long citizenId;
    private long walkCount;
    private Integer walksDuration;
    private long dogCount;
    private long points;

    public StatisticsModel() {
    }

    public long getCitizenId() { return citizenId; }

    public void setCitizenId(long citizenId) { this.citizenId = citizenId; }

    public long getWalkCount() { return walkCount; }

    public void setWalkCount(long walkCount) { this.walkCount = walkCount; }

    public Integer getWalksDuration() { return walksDuration; }

    public void setWalksDuration(Integer walksDuration) { this.walksDuration = walksDuration; }

    public long getDogCount() { return dogCount; }

    public void setDogCount(long dogCount) { this.dogCount = dogCount; }

    public long getPoints() { return points; }

    public void setPoints(long points) { this.points = points; }

}

File: IzvorniKod/frontend/oneclick-frontend/src/models/statistics.model.ts
export default class StatisticsModel {
    citizenInitials: string;
    walkCount: number;
    walksDuration: number;
    dogCount: number;
    points: number;

    constructor(citizenInitials: string, walkCount: number, walksDuration: number, dogCount: number, points: number) {
        this.citizenInitials = citizenInitials;
        this.walkCount = walkCount;
        this.walksDuration = walksDuration;
        this.dogCount = dogCount;
        this.points = points;
    }
}